hummingbot/connector/exchange/paper_trade/paper_trade_exchange.pyx

```diff
@@ -906,7 +906,16 @@ cdef class PaperTradeExchange(ExchangeBase):
                           object order_side,
                           object amount,
                           object price):
-        return estimate_fee(self.name, order_type is OrderType.LIMIT)
+        return build_trade_fee(
+            self.name,
+            is_maker=order_type is OrderType.LIMIT,
```

Comment:
Should we consider LIMITMAKER also a maker order type?

Reply:
After looking into it, I don't think this one is necessary with the current implementation as we don't override `ExchangeBase.supported_order_types`, which is by default implemented as:
```
    def supported_order_types(self):
        return [OrderType.LIMIT, OrderType.MARKET]
```